src/mm-dropdown/mm-dropdown.js

```diff
@@ -87,7 +72,22 @@
 				type: Boolean,
 				reflectToAttribute: true
 			},
-			_layout: String
+			layout: String,
```

Comment:
i dont think this is valid syntax. it will work, but what its doing is essentially setting layout to String.prototype